test/javascript/tests/changes.js

```diff
@@ -17,131 +17,140 @@ function jsonp(obj) {
 }
 
 couchTests.changes = function(debug) {
-  var db = new CouchDB("test_suite_db", {"X-Couch-Full-Commit":"true"});
-  db.deleteDb();
-  db.createDb();
+  var db;
   if (debug) debugger;
 
-  var req = CouchDB.request("GET", "/test_suite_db/_changes");
-  var resp = JSON.parse(req.responseText);
+  // poor man's browser detection
```

Comment:
Not sure if this is ever need since we drop feature to run the test suite from browser.